docs/patterns/fabric.rst

```diff
@@ -156,6 +156,25 @@ location where it's expected (eg: :file:`/var/www/yourapplication`).
 Either way, in our case here we only expect one or two servers and we can
 upload them ahead of time by hand.
 
+Configuring egg_info
+--------------------
+If you need to configure your fabric build with tags, you can create a `setup.cfg`
+file in the root of your app. An example would be:
+
+    [egg_info]
+    tag_svn_revision = 1
```

Comment:
Yeah, `tag_svn_revision` only really works with svn.  'dev<date>` is generally useful.
```
tag_build = .dev
tag_date = 1
```